<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="10" ignored="10" total="44" passed="19" failed="5" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-10-05T00:46:22 EDT" name="Suite" finished-at="2023-10-05T00:48:25 EDT" duration-ms="122598">
    <groups>
      <group name="possitive">
        <method signature="Draft.draftfail()[pri:0, instance:com.tutorialNinja.functionalTests.Draft@31190526]" name="draftfail" class="com.tutorialNinja.functionalTests.Draft"/>
        <method signature="R1dot1_TS002_LoginFunctionality.r1dot1_TS002_TC001()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality@2aceadd4]" name="r1dot1_TS002_TC001" class="com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality"/>
        <method signature="R1dot2_TS006_ProductCompareFunctionality.r1dot2_TS006_TC001()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot2_TS006_ProductCompareFunctionality@3a52dba3]" name="r1dot2_TS006_TC001" class="com.tutorialNinja.functionalTests.R1dot2_TS006_ProductCompareFunctionality"/>
        <method signature="R1dot1_TS001_RegisterFunctionality.r1dot1_TS001_TC001()[pri:1, instance:com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality@24269709]" name="r1dot1_TS001_TC001" class="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality"/>
        <method signature="R1dot1_TS003_LogoutFunctionality.r1dot1_TS003_TC001()[pri:1, instance:com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality@24aed80c]" name="r1dot1_TS003_TC001" class="com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality"/>
        <method signature="R1dot1_TS004_ForgottenPasswordFunctionality.r1dot1_TS004_TC001()[pri:1, instance:com.tutorialNinja.functionalTests.R1dot1_TS004_ForgottenPasswordFunctionality@279fedbd]" name="r1dot1_TS004_TC001" class="com.tutorialNinja.functionalTests.R1dot1_TS004_ForgottenPasswordFunctionality"/>
        <method signature="R1dot2_TS005_ProductSearchFunctionality.r1dot2_TS005_TC001()[pri:1, instance:com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality@2a5c8d3f]" name="r1dot2_TS005_TC001" class="com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality"/>
        <method signature="R1dot1_TS003_LogoutFunctionality.r1dot1_TS003_TC002()[pri:2, instance:com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality@24aed80c]" name="r1dot1_TS003_TC002" class="com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality"/>
        <method signature="R1dot1_TS004_ForgottenPasswordFunctionality.r1dot1_TS004_TC002()[pri:2, instance:com.tutorialNinja.functionalTests.R1dot1_TS004_ForgottenPasswordFunctionality@279fedbd]" name="r1dot1_TS004_TC002" class="com.tutorialNinja.functionalTests.R1dot1_TS004_ForgottenPasswordFunctionality"/>
        <method signature="R1dot2_TS005_ProductSearchFunctionality.R1dot2_TS005_TC024(java.lang.String)[pri:2, instance:com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality@2a5c8d3f]" name="R1dot2_TS005_TC024" class="com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality"/>
        <method signature="R1dot1_TS001_RegisterFunctionality.r1dot1_TS001_TC003()[pri:3, instance:com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality@24269709]" name="r1dot1_TS001_TC003" class="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality"/>
        <method signature="R1dot1_TS003_LogoutFunctionality.r1dot1_TS003_TC003()[pri:3, instance:com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality@24aed80c]" name="r1dot1_TS003_TC003" class="com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality"/>
      </group> <!-- possitive -->
      <group name="negetive">
        <method signature="Draft.loginPagefail()[pri:0, instance:com.tutorialNinja.functionalTests.Draft@31190526]" name="loginPagefail" class="com.tutorialNinja.functionalTests.Draft"/>
        <method signature="R1dot3_TS012_CheckoutFunctionality.R1dot3_TS012_TC001()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot3_TS012_CheckoutFunctionality@752325ad]" name="R1dot3_TS012_TC001" class="com.tutorialNinja.functionalTests.R1dot3_TS012_CheckoutFunctionality"/>
        <method signature="R1dot1_TS002_LoginFunctionality.r1dot1_TS002_TC002()[pri:1, instance:com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality@2aceadd4]" name="r1dot1_TS002_TC002" class="com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality"/>
        <method signature="R1dot1_TS002_LoginFunctionality.r1dot1_TS002_TC003()[pri:2, instance:com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality@2aceadd4]" name="r1dot1_TS002_TC003" class="com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality"/>
        <method signature="R1dot1_TS001_RegisterFunctionality.r1dot1_TS001_TC004()[pri:4, instance:com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality@24269709]" name="r1dot1_TS001_TC004" class="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality"/>
        <method signature="R1dot1_TS001_RegisterFunctionality.r1dot1_TS001_TC028()[pri:28, instance:com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality@24269709]" name="r1dot1_TS001_TC028" class="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality"/>
      </group> <!-- negetive -->
      <group name="smoke">
        <method signature="R1dot1_TS002_LoginFunctionality.r1dot1_TS002_TC001()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality@2aceadd4]" name="r1dot1_TS002_TC001" class="com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality"/>
        <method signature="R1dot1_TS001_RegisterFunctionality.r1dot1_TS001_TC001()[pri:1, instance:com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality@24269709]" name="r1dot1_TS001_TC001" class="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality"/>
        <method signature="R1dot1_TS002_LoginFunctionality.r1dot1_TS002_TC002()[pri:1, instance:com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality@2aceadd4]" name="r1dot1_TS002_TC002" class="com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality"/>
        <method signature="R1dot1_TS003_LogoutFunctionality.r1dot1_TS003_TC001()[pri:1, instance:com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality@24aed80c]" name="r1dot1_TS003_TC001" class="com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality"/>
        <method signature="R1dot1_TS004_ForgottenPasswordFunctionality.r1dot1_TS004_TC001()[pri:1, instance:com.tutorialNinja.functionalTests.R1dot1_TS004_ForgottenPasswordFunctionality@279fedbd]" name="r1dot1_TS004_TC001" class="com.tutorialNinja.functionalTests.R1dot1_TS004_ForgottenPasswordFunctionality"/>
        <method signature="R1dot2_TS005_ProductSearchFunctionality.r1dot2_TS005_TC001()[pri:1, instance:com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality@2a5c8d3f]" name="r1dot2_TS005_TC001" class="com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality"/>
        <method signature="R1dot1_TS002_LoginFunctionality.r1dot1_TS002_TC003()[pri:2, instance:com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality@2aceadd4]" name="r1dot1_TS002_TC003" class="com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality"/>
        <method signature="R1dot1_TS003_LogoutFunctionality.r1dot1_TS003_TC002()[pri:2, instance:com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality@24aed80c]" name="r1dot1_TS003_TC002" class="com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality"/>
        <method signature="R1dot1_TS004_ForgottenPasswordFunctionality.r1dot1_TS004_TC002()[pri:2, instance:com.tutorialNinja.functionalTests.R1dot1_TS004_ForgottenPasswordFunctionality@279fedbd]" name="r1dot1_TS004_TC002" class="com.tutorialNinja.functionalTests.R1dot1_TS004_ForgottenPasswordFunctionality"/>
        <method signature="R1dot2_TS005_ProductSearchFunctionality.R1dot2_TS005_TC024(java.lang.String)[pri:2, instance:com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality@2a5c8d3f]" name="R1dot2_TS005_TC024" class="com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality"/>
        <method signature="R1dot1_TS001_RegisterFunctionality.r1dot1_TS001_TC003()[pri:3, instance:com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality@24269709]" name="r1dot1_TS001_TC003" class="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality"/>
        <method signature="R1dot1_TS003_LogoutFunctionality.r1dot1_TS003_TC003()[pri:3, instance:com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality@24aed80c]" name="r1dot1_TS003_TC003" class="com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality"/>
        <method signature="R1dot1_TS001_RegisterFunctionality.r1dot1_TS001_TC004()[pri:4, instance:com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality@24269709]" name="r1dot1_TS001_TC004" class="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality"/>
        <method signature="R1dot1_TS001_RegisterFunctionality.r1dot1_TS001_TC028()[pri:28, instance:com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality@24269709]" name="r1dot1_TS001_TC028" class="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality"/>
      </group> <!-- smoke -->
    </groups>
    <test started-at="2023-10-05T00:46:22 EDT" name="Tutorial Ninja functional Regression Test" finished-at="2023-10-05T00:48:25 EDT" duration-ms="122598">
      <class name="com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality">
        <test-method signature="r1dot1_TS003_TC001()[pri:1, instance:com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality@24aed80c]" started-at="2023-10-05T00:46:37 EDT" name="r1dot1_TS003_TC001" description="R1.1_TS003_TC001 Verify Logging out by selecting Logout option from &#039;My Account&#039; drop menu" finished-at="2023-10-05T00:46:49 EDT" duration-ms="12351" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- r1dot1_TS003_TC001 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality@24aed80c]" started-at="2023-10-05T00:46:49 EDT" name="quitDriver" finished-at="2023-10-05T00:46:49 EDT" duration-ms="219" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method signature="r1dot1_TS003_TC002()[pri:2, instance:com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality@24aed80c]" started-at="2023-10-05T00:46:49 EDT" name="r1dot1_TS003_TC002" description="R1.1_TS003_TC002 Verify Logging out by selecting Logout option from &#039;Right Column&#039; options" finished-at="2023-10-05T00:47:00 EDT" duration-ms="10775" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- r1dot1_TS003_TC002 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality@24aed80c]" started-at="2023-10-05T00:47:00 EDT" name="quitDriver" finished-at="2023-10-05T00:47:01 EDT" duration-ms="802" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method retried="true" signature="r1dot1_TS003_TC003()[pri:3, instance:com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality@24aed80c]" started-at="2023-10-05T00:47:01 EDT" name="r1dot1_TS003_TC003" description="R1.1_TS003_TC003 Verify the Application session status, after logging and closing the Browser without logging out" finished-at="2023-10-05T00:47:30 EDT" duration-ms="29372" status="SKIP">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[no such element: Unable to locate element: {"method":"xpath","selector":"//ul[@class='dropdown-menu dropdown-menu-right']//a[text()='My Account']"}
  (Session info: chrome=117.0.5938.132)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.12.1', revision: '8e34639b11'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.7'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [86ed432d4e349ff9a844e111b337041f, findElement {using=xpath, value=//ul[@class='dropdown-menu dropdown-menu-right']//a[text()='My Account']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 117.0.5938.132, chrome: {chromedriverVersion: 117.0.5938.149 (e3344ddefa1..., userDataDir: C:\Users\naimn\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:51470}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:51470/devtoo..., se:cdpVersion: 117.0.5938.132, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 86ed432d4e349ff9a844e111b337041f]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//ul[@class='dropdown-menu dropdown-menu-right']//a[text()='My Account']"}
  (Session info: chrome=117.0.5938.132)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.12.1', revision: '8e34639b11'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.7'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [86ed432d4e349ff9a844e111b337041f, findElement {using=xpath, value=//ul[@class='dropdown-menu dropdown-menu-right']//a[text()='My Account']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 117.0.5938.132, chrome: {chromedriverVersion: 117.0.5938.149 (e3344ddefa1..., userDataDir: C:\Users\naimn\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:51470}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:51470/devtoo..., se:cdpVersion: 117.0.5938.132, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 86ed432d4e349ff9a844e111b337041f
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:52)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:191)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:196)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:171)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:602)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:59)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:359)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:353)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:68)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at jdk.proxy2/jdk.proxy2.$Proxy17.click(Unknown Source)
at com.tutorialNinja.pageObjectFactory.CommonElements.click_MyAccount(CommonElements.java:84)
at com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality.r1dot1_TS003_TC003(R1dot1_TS003_LogoutFunctionality.java:60)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- r1dot1_TS003_TC003 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality@24aed80c]" started-at="2023-10-05T00:47:30 EDT" name="quitDriver" finished-at="2023-10-05T00:47:30 EDT" duration-ms="232" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method retried="true" signature="r1dot1_TS003_TC003()[pri:3, instance:com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality@24aed80c]" started-at="2023-10-05T00:47:30 EDT" name="r1dot1_TS003_TC003" description="R1.1_TS003_TC003 Verify the Application session status, after logging and closing the Browser without logging out" finished-at="2023-10-05T00:47:57 EDT" duration-ms="26411" status="SKIP">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[no such element: Unable to locate element: {"method":"xpath","selector":"//ul[@class='dropdown-menu dropdown-menu-right']//a[text()='My Account']"}
  (Session info: chrome=117.0.5938.132)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.12.1', revision: '8e34639b11'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.7'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [35137720100261b55256e724f23929eb, findElement {using=xpath, value=//ul[@class='dropdown-menu dropdown-menu-right']//a[text()='My Account']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 117.0.5938.132, chrome: {chromedriverVersion: 117.0.5938.149 (e3344ddefa1..., userDataDir: C:\Users\naimn\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:51668}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:51668/devtoo..., se:cdpVersion: 117.0.5938.132, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 35137720100261b55256e724f23929eb]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//ul[@class='dropdown-menu dropdown-menu-right']//a[text()='My Account']"}
  (Session info: chrome=117.0.5938.132)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.12.1', revision: '8e34639b11'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.7'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [35137720100261b55256e724f23929eb, findElement {using=xpath, value=//ul[@class='dropdown-menu dropdown-menu-right']//a[text()='My Account']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 117.0.5938.132, chrome: {chromedriverVersion: 117.0.5938.149 (e3344ddefa1..., userDataDir: C:\Users\naimn\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:51668}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:51668/devtoo..., se:cdpVersion: 117.0.5938.132, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 35137720100261b55256e724f23929eb
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:52)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:191)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:196)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:171)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:602)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:59)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:359)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:353)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:68)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at jdk.proxy2/jdk.proxy2.$Proxy17.click(Unknown Source)
at com.tutorialNinja.pageObjectFactory.CommonElements.click_MyAccount(CommonElements.java:84)
at com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality.r1dot1_TS003_TC003(R1dot1_TS003_LogoutFunctionality.java:60)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:248)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- r1dot1_TS003_TC003 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality@24aed80c]" started-at="2023-10-05T00:47:57 EDT" name="quitDriver" finished-at="2023-10-05T00:47:57 EDT" duration-ms="184" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method signature="r1dot1_TS003_TC003()[pri:3, instance:com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality@24aed80c]" started-at="2023-10-05T00:47:57 EDT" name="r1dot1_TS003_TC003" description="R1.1_TS003_TC003 Verify the Application session status, after logging and closing the Browser without logging out" finished-at="2023-10-05T00:48:24 EDT" duration-ms="27404" status="FAIL">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[no such element: Unable to locate element: {"method":"xpath","selector":"//ul[@class='dropdown-menu dropdown-menu-right']//a[text()='My Account']"}
  (Session info: chrome=117.0.5938.132)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.12.1', revision: '8e34639b11'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.7'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [7ea78bbf43096dffb725f170ec52396c, findElement {using=xpath, value=//ul[@class='dropdown-menu dropdown-menu-right']//a[text()='My Account']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 117.0.5938.132, chrome: {chromedriverVersion: 117.0.5938.149 (e3344ddefa1..., userDataDir: C:\Users\naimn\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:51736}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:51736/devtoo..., se:cdpVersion: 117.0.5938.132, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 7ea78bbf43096dffb725f170ec52396c]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//ul[@class='dropdown-menu dropdown-menu-right']//a[text()='My Account']"}
  (Session info: chrome=117.0.5938.132)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.12.1', revision: '8e34639b11'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.7'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [7ea78bbf43096dffb725f170ec52396c, findElement {using=xpath, value=//ul[@class='dropdown-menu dropdown-menu-right']//a[text()='My Account']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 117.0.5938.132, chrome: {chromedriverVersion: 117.0.5938.149 (e3344ddefa1..., userDataDir: C:\Users\naimn\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:51736}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:51736/devtoo..., se:cdpVersion: 117.0.5938.132, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 7ea78bbf43096dffb725f170ec52396c
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:52)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:191)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:196)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:171)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:602)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:59)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:359)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:353)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:68)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at jdk.proxy2/jdk.proxy2.$Proxy17.click(Unknown Source)
at com.tutorialNinja.pageObjectFactory.CommonElements.click_MyAccount(CommonElements.java:84)
at com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality.r1dot1_TS003_TC003(R1dot1_TS003_LogoutFunctionality.java:60)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:248)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- r1dot1_TS003_TC003 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality@24aed80c]" started-at="2023-10-05T00:48:25 EDT" name="quitDriver" finished-at="2023-10-05T00:48:25 EDT" duration-ms="274" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
      </class> <!-- com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality -->
      <class name="com.tutorialNinja.functionalTests.Draft">
        <test-method retried="true" signature="draftfail()[pri:0, instance:com.tutorialNinja.functionalTests.Draft@31190526]" started-at="2023-10-05T00:46:23 EDT" name="draftfail" description="negetive test case draftfail" finished-at="2023-10-05T00:46:36 EDT" duration-ms="12828" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1067)
at org.testng.Assert.assertEquals(Assert.java:1031)
at org.testng.Assert.assertEquals(Assert.java:1077)
at com.tutorialNinja.functionalTests.Draft.draftfail(Draft.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- draftfail -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.Draft@31190526]" started-at="2023-10-05T00:46:36 EDT" name="quitDriver" finished-at="2023-10-05T00:46:37 EDT" duration-ms="917" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method retried="true" signature="draftfail()[pri:0, instance:com.tutorialNinja.functionalTests.Draft@31190526]" started-at="2023-10-05T00:46:37 EDT" name="draftfail" description="negetive test case draftfail" finished-at="2023-10-05T00:46:44 EDT" duration-ms="7860" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1067)
at org.testng.Assert.assertEquals(Assert.java:1031)
at org.testng.Assert.assertEquals(Assert.java:1077)
at com.tutorialNinja.functionalTests.Draft.draftfail(Draft.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:248)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- draftfail -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.Draft@31190526]" started-at="2023-10-05T00:46:44 EDT" name="quitDriver" finished-at="2023-10-05T00:46:45 EDT" duration-ms="362" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method signature="draftfail()[pri:0, instance:com.tutorialNinja.functionalTests.Draft@31190526]" started-at="2023-10-05T00:46:45 EDT" name="draftfail" description="negetive test case draftfail" finished-at="2023-10-05T00:46:52 EDT" duration-ms="6967" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1067)
at org.testng.Assert.assertEquals(Assert.java:1031)
at org.testng.Assert.assertEquals(Assert.java:1077)
at com.tutorialNinja.functionalTests.Draft.draftfail(Draft.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:248)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- draftfail -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.Draft@31190526]" started-at="2023-10-05T00:46:52 EDT" name="quitDriver" finished-at="2023-10-05T00:46:53 EDT" duration-ms="1224" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method retried="true" signature="loginPagefail()[pri:0, instance:com.tutorialNinja.functionalTests.Draft@31190526]" started-at="2023-10-05T00:46:53 EDT" name="loginPagefail" description="negetive test case loginPagefail" finished-at="2023-10-05T00:47:00 EDT" duration-ms="6401" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Customer] but found [Account Login]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Customer] but found [Account Login]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at com.tutorialNinja.functionalTests.Draft.loginPagefail(Draft.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPagefail -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.Draft@31190526]" started-at="2023-10-05T00:47:00 EDT" name="quitDriver" finished-at="2023-10-05T00:47:01 EDT" duration-ms="650" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method retried="true" signature="loginPagefail()[pri:0, instance:com.tutorialNinja.functionalTests.Draft@31190526]" started-at="2023-10-05T00:47:01 EDT" name="loginPagefail" description="negetive test case loginPagefail" finished-at="2023-10-05T00:47:07 EDT" duration-ms="6784" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Customer] but found [Account Login]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Customer] but found [Account Login]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at com.tutorialNinja.functionalTests.Draft.loginPagefail(Draft.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:248)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPagefail -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.Draft@31190526]" started-at="2023-10-05T00:47:07 EDT" name="quitDriver" finished-at="2023-10-05T00:47:08 EDT" duration-ms="232" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method signature="loginPagefail()[pri:0, instance:com.tutorialNinja.functionalTests.Draft@31190526]" started-at="2023-10-05T00:47:08 EDT" name="loginPagefail" description="negetive test case loginPagefail" finished-at="2023-10-05T00:47:15 EDT" duration-ms="7621" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Customer] but found [Account Login]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Customer] but found [Account Login]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at com.tutorialNinja.functionalTests.Draft.loginPagefail(Draft.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:248)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPagefail -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.Draft@31190526]" started-at="2023-10-05T00:47:16 EDT" name="quitDriver" finished-at="2023-10-05T00:47:16 EDT" duration-ms="615" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method signature="readExcelDraft()[pri:0, instance:com.tutorialNinja.functionalTests.Draft@31190526]" started-at="2023-10-05T00:47:16 EDT" name="readExcelDraft" description="validate readExcelDraft" finished-at="2023-10-05T00:47:18 EDT" duration-ms="2069" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- readExcelDraft -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.Draft@31190526]" started-at="2023-10-05T00:47:18 EDT" name="quitDriver" finished-at="2023-10-05T00:47:18 EDT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method signature="testHomePageNav()[pri:0, instance:com.tutorialNinja.functionalTests.Draft@31190526]" started-at="2023-10-05T00:47:18 EDT" name="testHomePageNav" description="draft test of product catagory page" finished-at="2023-10-05T00:47:31 EDT" duration-ms="13184" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testHomePageNav -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.Draft@31190526]" started-at="2023-10-05T00:47:31 EDT" name="quitDriver" finished-at="2023-10-05T00:47:33 EDT" duration-ms="1423" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method signature="testHomepageFeaturedProduct()[pri:0, instance:com.tutorialNinja.functionalTests.Draft@31190526]" started-at="2023-10-05T00:47:33 EDT" name="testHomepageFeaturedProduct" description="click on homepage featured product" finished-at="2023-10-05T00:47:44 EDT" duration-ms="11509" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testHomepageFeaturedProduct -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.Draft@31190526]" started-at="2023-10-05T00:47:44 EDT" name="quitDriver" finished-at="2023-10-05T00:47:45 EDT" duration-ms="144" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
      </class> <!-- com.tutorialNinja.functionalTests.Draft -->
      <class name="com.tutorialNinja.functionalTests.R1dot3_TS012_CheckoutFunctionality">
        <test-method signature="R1dot3_TS012_TC001()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot3_TS012_CheckoutFunctionality@752325ad]" started-at="2023-10-05T00:46:23 EDT" name="R1dot3_TS012_TC001" description="R1.3_TS012_TC001 As a user I shouldn&#039;t be able to  navigate to Checkout page when there is no product in the Shopping Cart" finished-at="2023-10-05T00:46:35 EDT" duration-ms="12518" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- R1dot3_TS012_TC001 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot3_TS012_CheckoutFunctionality@752325ad]" started-at="2023-10-05T00:46:35 EDT" name="quitDriver" finished-at="2023-10-05T00:46:37 EDT" duration-ms="1144" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
      </class> <!-- com.tutorialNinja.functionalTests.R1dot3_TS012_CheckoutFunctionality -->
      <class name="com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality">
        <test-method signature="r1dot1_TS002_TC001()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality@2aceadd4]" started-at="2023-10-05T00:46:23 EDT" name="r1dot1_TS002_TC001" description="R1.1_TS002_TC001 Verify logging into the Application using valid credentials" finished-at="2023-10-05T00:46:38 EDT" duration-ms="15332" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- r1dot1_TS002_TC001 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality@2aceadd4]" started-at="2023-10-05T00:46:38 EDT" name="quitDriver" finished-at="2023-10-05T00:46:40 EDT" duration-ms="2021" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method retried="true" signature="r1dot1_TS002_TC002()[pri:1, instance:com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality@2aceadd4]" started-at="2023-10-05T00:46:40 EDT" name="r1dot1_TS002_TC002" description="R1.1_TS002_TC002 Verify logging into the Application using invalid credentials (i.e. Invalid email address and Invalid Password)" finished-at="2023-10-05T00:46:51 EDT" duration-ms="10778" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality.r1dot1_TS002_TC002(R1dot1_TS002_LoginFunctionality.java:43)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- r1dot1_TS002_TC002 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality@2aceadd4]" started-at="2023-10-05T00:46:51 EDT" name="quitDriver" finished-at="2023-10-05T00:46:52 EDT" duration-ms="1198" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method retried="true" signature="r1dot1_TS002_TC002()[pri:1, instance:com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality@2aceadd4]" started-at="2023-10-05T00:46:52 EDT" name="r1dot1_TS002_TC002" description="R1.1_TS002_TC002 Verify logging into the Application using invalid credentials (i.e. Invalid email address and Invalid Password)" finished-at="2023-10-05T00:47:03 EDT" duration-ms="10921" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality.r1dot1_TS002_TC002(R1dot1_TS002_LoginFunctionality.java:43)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:248)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- r1dot1_TS002_TC002 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality@2aceadd4]" started-at="2023-10-05T00:47:03 EDT" name="quitDriver" finished-at="2023-10-05T00:47:03 EDT" duration-ms="250" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method signature="r1dot1_TS002_TC002()[pri:1, instance:com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality@2aceadd4]" started-at="2023-10-05T00:47:03 EDT" name="r1dot1_TS002_TC002" description="R1.1_TS002_TC002 Verify logging into the Application using invalid credentials (i.e. Invalid email address and Invalid Password)" finished-at="2023-10-05T00:47:13 EDT" duration-ms="9274" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality.r1dot1_TS002_TC002(R1dot1_TS002_LoginFunctionality.java:43)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:248)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- r1dot1_TS002_TC002 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality@2aceadd4]" started-at="2023-10-05T00:47:13 EDT" name="quitDriver" finished-at="2023-10-05T00:47:13 EDT" duration-ms="239" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method retried="true" signature="r1dot1_TS002_TC003()[pri:2, instance:com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality@2aceadd4]" started-at="2023-10-05T00:47:13 EDT" name="r1dot1_TS002_TC003" description="R1.1_TS002_TC003 Verify logging into the Application using invalid email address and valid Password" finished-at="2023-10-05T00:47:22 EDT" duration-ms="9244" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality.r1dot1_TS002_TC003(R1dot1_TS002_LoginFunctionality.java:58)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- r1dot1_TS002_TC003 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality@2aceadd4]" started-at="2023-10-05T00:47:22 EDT" name="quitDriver" finished-at="2023-10-05T00:47:23 EDT" duration-ms="191" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method retried="true" signature="r1dot1_TS002_TC003()[pri:2, instance:com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality@2aceadd4]" started-at="2023-10-05T00:47:23 EDT" name="r1dot1_TS002_TC003" description="R1.1_TS002_TC003 Verify logging into the Application using invalid email address and valid Password" finished-at="2023-10-05T00:47:31 EDT" duration-ms="7977" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality.r1dot1_TS002_TC003(R1dot1_TS002_LoginFunctionality.java:58)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:248)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- r1dot1_TS002_TC003 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality@2aceadd4]" started-at="2023-10-05T00:47:31 EDT" name="quitDriver" finished-at="2023-10-05T00:47:31 EDT" duration-ms="206" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method signature="r1dot1_TS002_TC003()[pri:2, instance:com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality@2aceadd4]" started-at="2023-10-05T00:47:31 EDT" name="r1dot1_TS002_TC003" description="R1.1_TS002_TC003 Verify logging into the Application using invalid email address and valid Password" finished-at="2023-10-05T00:47:42 EDT" duration-ms="11330" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality.r1dot1_TS002_TC003(R1dot1_TS002_LoginFunctionality.java:58)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:248)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- r1dot1_TS002_TC003 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality@2aceadd4]" started-at="2023-10-05T00:47:42 EDT" name="quitDriver" finished-at="2023-10-05T00:47:43 EDT" duration-ms="319" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
      </class> <!-- com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality -->
      <class name="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality">
        <test-method signature="r1dot1_TS001_TC001()[pri:1, instance:com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality@24269709]" started-at="2023-10-05T00:46:23 EDT" name="r1dot1_TS001_TC001" description="R1.1_TS001_TC001 As a user I should be able to Register an Account by providing only the Mandatory fields" finished-at="2023-10-05T00:46:41 EDT" duration-ms="17774" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- r1dot1_TS001_TC001 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality@24269709]" started-at="2023-10-05T00:46:41 EDT" name="quitDriver" finished-at="2023-10-05T00:46:42 EDT" duration-ms="972" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method signature="r1dot1_TS001_TC003()[pri:3, instance:com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality@24269709]" started-at="2023-10-05T00:46:42 EDT" name="r1dot1_TS001_TC003" description="R1.1_TS001_TC003 Verify Registering an Account by providing all the fields" finished-at="2023-10-05T00:46:56 EDT" duration-ms="14524" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- r1dot1_TS001_TC003 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality@24269709]" started-at="2023-10-05T00:46:56 EDT" name="quitDriver" finished-at="2023-10-05T00:46:56 EDT" duration-ms="288" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method signature="r1dot1_TS001_TC004()[pri:4, instance:com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality@24269709]" started-at="2023-10-05T00:46:56 EDT" name="r1dot1_TS001_TC004" description="R1.1_TS001_TC004 Verify proper error messages are displayed for the mandatory fields, when you don&#039;t provide any mandatory fields in the &#039;Register Account&#039; page and submit" finished-at="2023-10-05T00:47:07 EDT" duration-ms="10198" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- r1dot1_TS001_TC004 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality@24269709]" started-at="2023-10-05T00:47:07 EDT" name="quitDriver" finished-at="2023-10-05T00:47:07 EDT" duration-ms="252" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method signature="r1dot1_TS001_TC028()[pri:28, instance:com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality@24269709]" started-at="2023-10-05T00:47:07 EDT" name="r1dot1_TS001_TC028" description="R1.1_TS001_TC028 Verify the application throws proper error message when user click continue button without selecting privacy and policy check box." finished-at="2023-10-05T00:47:18 EDT" duration-ms="11336" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- r1dot1_TS001_TC028 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality@24269709]" started-at="2023-10-05T00:47:18 EDT" name="quitDriver" finished-at="2023-10-05T00:47:19 EDT" duration-ms="378" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
      </class> <!-- com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality -->
      <class name="com.tutorialNinja.functionalTests.R1dot1_TS004_ForgottenPasswordFunctionality">
        <test-method signature="r1dot1_TS004_TC001()[pri:1, instance:com.tutorialNinja.functionalTests.R1dot1_TS004_ForgottenPasswordFunctionality@279fedbd]" started-at="2023-10-05T00:46:38 EDT" name="r1dot1_TS004_TC001" description="R1.1_TS004_TC001 as a Registered user I should be able to reset my password by using &quot;Forgotten Password&quot; link in &quot;Returning Customer&quot; Login box" finished-at="2023-10-05T00:46:50 EDT" duration-ms="11488" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- r1dot1_TS004_TC001 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot1_TS004_ForgottenPasswordFunctionality@279fedbd]" started-at="2023-10-05T00:46:50 EDT" name="quitDriver" finished-at="2023-10-05T00:46:50 EDT" duration-ms="310" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method signature="r1dot1_TS004_TC002()[pri:2, instance:com.tutorialNinja.functionalTests.R1dot1_TS004_ForgottenPasswordFunctionality@279fedbd]" started-at="2023-10-05T00:46:50 EDT" name="r1dot1_TS004_TC002" description="R1.1_TS004_TC002 as a Registered user I should be able to reset my password by using &quot;Forgotten Password&quot; link in right column" finished-at="2023-10-05T00:47:03 EDT" duration-ms="13110" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- r1dot1_TS004_TC002 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot1_TS004_ForgottenPasswordFunctionality@279fedbd]" started-at="2023-10-05T00:47:03 EDT" name="quitDriver" finished-at="2023-10-05T00:47:04 EDT" duration-ms="626" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
      </class> <!-- com.tutorialNinja.functionalTests.R1dot1_TS004_ForgottenPasswordFunctionality -->
      <class name="com.tutorialNinja.functionalTests.R1dot2_TS006_ProductCompareFunctionality">
        <test-method signature="r1dot2_TS006_TC001()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot2_TS006_ProductCompareFunctionality@3a52dba3]" started-at="2023-10-05T00:46:23 EDT" name="r1dot2_TS006_TC001" description="R1.2_TS006_TC001 As an user I should be able to select products for comparison." finished-at="2023-10-05T00:46:38 EDT" duration-ms="15128" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- r1dot2_TS006_TC001 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot2_TS006_ProductCompareFunctionality@3a52dba3]" started-at="2023-10-05T00:46:38 EDT" name="quitDriver" finished-at="2023-10-05T00:46:38 EDT" duration-ms="282" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
      </class> <!-- com.tutorialNinja.functionalTests.R1dot2_TS006_ProductCompareFunctionality -->
      <class name="com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality">
        <test-method signature="r1dot2_TS005_TC001()[pri:1, instance:com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality@2a5c8d3f]" started-at="2023-10-05T00:47:04 EDT" name="r1dot2_TS005_TC001" description="R1.2_TS005_TC001 As an user I should be able to search a product which is present in database." finished-at="2023-10-05T00:47:15 EDT" duration-ms="11555" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- r1dot2_TS005_TC001 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality@2a5c8d3f]" started-at="2023-10-05T00:47:15 EDT" name="quitDriver" finished-at="2023-10-05T00:47:16 EDT" duration-ms="847" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method signature="R1dot2_TS005_TC024(java.lang.String)[pri:2, instance:com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality@2a5c8d3f]" started-at="2023-10-05T00:47:16 EDT" name="R1dot2_TS005_TC024" description="R1.2_TS005_TC024 Verify user can select a product from featured product list from HomePage" data-provider="productList" finished-at="2023-10-05T00:47:24 EDT" duration-ms="7832" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[MacBook]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- R1dot2_TS005_TC024 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality@2a5c8d3f]" started-at="2023-10-05T00:47:24 EDT" name="quitDriver" finished-at="2023-10-05T00:47:24 EDT" duration-ms="227" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method signature="R1dot2_TS005_TC024(java.lang.String)[pri:2, instance:com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality@2a5c8d3f]" started-at="2023-10-05T00:47:24 EDT" name="R1dot2_TS005_TC024" description="R1.2_TS005_TC024 Verify user can select a product from featured product list from HomePage" data-provider="productList" finished-at="2023-10-05T00:47:32 EDT" duration-ms="7616" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[iPhone]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- R1dot2_TS005_TC024 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality@2a5c8d3f]" started-at="2023-10-05T00:47:32 EDT" name="quitDriver" finished-at="2023-10-05T00:47:33 EDT" duration-ms="1190" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method signature="R1dot2_TS005_TC024(java.lang.String)[pri:2, instance:com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality@2a5c8d3f]" started-at="2023-10-05T00:47:33 EDT" name="R1dot2_TS005_TC024" description="R1.2_TS005_TC024 Verify user can select a product from featured product list from HomePage" data-provider="productList" finished-at="2023-10-05T00:47:49 EDT" duration-ms="16095" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Apple Cinema 30"]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- R1dot2_TS005_TC024 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality@2a5c8d3f]" started-at="2023-10-05T00:47:49 EDT" name="quitDriver" finished-at="2023-10-05T00:47:49 EDT" duration-ms="173" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method signature="R1dot2_TS005_TC024(java.lang.String)[pri:2, instance:com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality@2a5c8d3f]" started-at="2023-10-05T00:47:49 EDT" name="R1dot2_TS005_TC024" description="R1.2_TS005_TC024 Verify user can select a product from featured product list from HomePage" data-provider="productList" finished-at="2023-10-05T00:47:57 EDT" duration-ms="8118" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Canon EOS 5D]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- R1dot2_TS005_TC024 -->
        <test-method is-config="true" signature="quitDriver()[pri:0, instance:com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality@2a5c8d3f]" started-at="2023-10-05T00:47:57 EDT" name="quitDriver" finished-at="2023-10-05T00:47:58 EDT" duration-ms="194" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
      </class> <!-- com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality -->
    </test> <!-- Tutorial Ninja functional Regression Test -->
  </suite> <!-- Suite -->
</testng-results>
